src/Tutor.Core/DomainModel/Feedback/EmotionsFeedback.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Tutor.Core.DomainModel.Feedback
+{
+    public class EmotionsFeedback
+    {
+        public int Id { get; set; }
+        public string Comment { get; set; }
```

Comment:
Can this be any string or is it limited to a handful of options?

Reply:
We expect text that's 2 or 3 sentences long here. 
Therefore, this should be enough.